src/main/java/software/amazon/event/kafkaconnector/EventBridgeEventId.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package software.amazon.event.kafkaconnector;
+
+import software.amazon.awssdk.services.eventbridge.model.PutEventsResultEntry;
+
+public class EventBridgeEventId {
```

Comment:
Question: we only use that for success records, right?

Reply:
Yes, the value class is used to give it a type. Maybe a `record` could be better than a `class`.

Reply:
✔️